main.js

```diff
@@ -53,6 +59,7 @@ define(function (require, exports, module) {
     var _preferences,
         _command,
         _styleTag,
+        _styleInline,
```

Comment:
I would have the tag removed via `unloadStyle()` like it does already, but I was unable to get it to work.